src/baldr/CMakeLists.txt

```diff
@@ -25,12 +25,25 @@ add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/date_time_windows_zones.h
 
 file(GLOB headers ${VALHALLA_SOURCE_DIR}/valhalla/baldr/*.h)
 
+# Builds a checksum of the traffictile header and store it in `traffic_tile_version.h`
+set(GENERATED_VERSION_HEADER "${CMAKE_CURRENT_BINARY_DIR}/traffic_tile_version.h")
+message("Generating traffictile version header")
+configure_file(${VALHALLA_SOURCE_DIR}/valhalla/baldr/traffictile.h
```

Comment:
@danpat suggested a way to evaluate this entirely in cmake, with the hack here to implicitly add a dependency on the file to re-run checksum on changes. My first attempt without this hack only generated the checksum on first run.